docs/content/en/commands/hugo.md

```diff
@@ -70,17 +70,17 @@ hugo [flags]
 
 ### SEE ALSO
 
-* [hugo build](/commands/hugo_build/)	 - build builds your site
```

Comment:
The CLI docs is generated with `go run main.go gen doc --dir docs/content/en/commands`, something we typically do before we release.

As to this PR, it would be much easier to review it without all the docs changes. So could you revert the `docs` folder to `master` and force push the PR branch?

Reply:
Done!